### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by codewithed**
Completed the following tasks for the Hello Microverse project:

* Create a new branch according to the GitHub flow rules.
* Add a .gitignore file
* Add index.html file
* Add styles.css file

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
[the remark this answers is not in the document]
Comment (codewithed) [reply test.md:1]: Thank you very much for your clear and concise review 😃.

**Review by sumairq — APPROVED** [review]
### Hi @codewithed , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good Commit history
✔️ Linters are passing
✔️ Descriptive README
✔️ Descriptive PR 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codewithed** [comment]
Thanks for the feedback @sumairq 😃

File: test.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Lucas-Masaba) [reply test.md]: No problem

**Pull request merged**